handler/handler.go

```diff
@@ -0,0 +1,154 @@
+package handler
+
+import (
+	"encoding/json"
+	"log"
+	"main/repo"
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+)
+
+type ServiceInterface interface {
```

Comment:
`ServiceInterface` -> `Service`
The type is already an interface, no need to duplicate this in the name